### DISCUSSION TITLE
df_print YAML Option

### DISCUSSION THREAD

**Discussion by zief0002** [quarto-dev/quarto-cli#567] [Feature Requests]
Hi Quarto Dev Team

I would love to see a `df_print` (or in Quarto `df-print`) YAML option similar to what is available in RMD files.

**Comment by jjallaire** [comment]
We don't currently have this but its on our list to provide enhanced table output so hopefully something like this will be available soon.

**Comment by zief0002** [comment]
Awesome!!!

On Thu, Apr 7, 2022 at 7:31 AM J.J. Allaire ***@***.***>
wrote:

> We don't currently have this but its on our list to provide enhanced table
> output so hopefully something like this will be available soon.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/quarto-dev/quarto-cli/discussions/567#discussioncomment-2523302>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAPZXHFF4NBAO3TMSIXGKLDVD3IQXANCNFSM5SZGVYUA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
-- 

Andrew Zieffler
Senior Lecturer | Quantitative Methods in Education
Department of Educational Psychology | College of Education and Human
Development | http://www.cehd.umn.edu/edpsych/
University of Minnesota | umn.edu
http://www.datadreaming.org/

**Comment by jjallaire** [comment]
Note that you can currently emulate this behavior by calling functions directly, e.g.

```r
rmarkdown::paged_table(data)
```

```r
knitr::kable(data)
```

etc.

**Comment by zief0002** [comment]
Yep. I was hoping for the YAML option just to keep my class notes cleaner
(students see the functions to view the data only) rather than the
formatting of the HTML. I know I can do this in two code chunks, but
workflow-wise it is cleaner on my end with just one chunk. I am glad to
hear this is on the table and am already enamored of Quarto.
Thanks for all the great work!




On Thu, Apr 7, 2022 at 7:32 AM J.J. Allaire ***@***.***>
wrote:

> Note that you can currently emulate this behavior by calling functions
> directly, e.g.
>
> rmarkdown::paged_table(data)
>
> knitr::kable(data)
>
> etc.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/quarto-dev/quarto-cli/discussions/567#discussioncomment-2523315>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAPZXHAWX2NMAZZPUNLEA53VD3IWJANCNFSM5SZGVYUA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
-- 

Andrew Zieffler
Senior Lecturer | Quantitative Methods in Education
Department of Educational Psychology | College of Education and Human
Development | http://www.cehd.umn.edu/edpsych/
University of Minnesota | umn.edu
http://www.datadreaming.org/

**Comment by cderv** [comment]
Quarto now has a `df-print` YAML option: https://quarto.org/docs/computations/r.html#data-frames

**Comment by mcanouil** [comment]
Closing this as completed.
